packages/babel-types/package.json

```diff
@@ -14,6 +14,13 @@
     "directory": "packages/babel-types"
   },
   "main": "lib/index.js",
+  "typesVersions": {
+    ">=3.7": {
```

Comment:
should this be `<=3.7`? (to not override generated types for new TS)

Reply:
Oh good catch!

Reply:
Oh wait, this means that typings for 3.7 have never worked :thinking: 

@JLHwung Given this, maybe it's fine to just forget about them?

Reply:
```suggestion
    "<3.7": {
```